masq/src/commands/set_configuration_command.rs

```diff
@@ -63,11 +63,15 @@ const SET_CONFIGURATION_ABOUT: &str =
 const START_BLOCK_HELP: &str =
     "Ordinal number of the Ethereum block where scanning for transactions will start.";
 
+pub fn set_configurationify<'a>(shared_schema_arg: Arg<'a, 'a>) -> Arg<'a, 'a> {
+    shared_schema_arg.takes_value(true).required(false)
```

Comment:
I don't think the `.required(false)` hurts anything, but I also think it's not necessary.

Reply:
I've removed it. 👍